index.bs

```diff
@@ -323,6 +337,15 @@ A [=request=] has an associated
 <dfn for=request>Attribution Reporting eligibility</dfn> (an [=eligibility=]).
 Unless otherwise stated it is "<code>[=eligibility/unset=]</code>".
 
+A [=request=] has an associated
+<dfn for=request>trigger verification metadata</dfn> which is null or a [=struct=] with the following items:
+<dl dfn-for="request/trigger verification metadata">
```

Comment:
I would prefer that we define this struct as a top-level type instead of always requiring it to be accessed through the request field.